specutils/manipulation/utils.py

```diff
@@ -1,26 +1,92 @@
 import numpy as np
+import warnings
 
+from astropy.utils.exceptions import AstropyUserWarning
 from ..spectra import Spectrum1D, SpectralRegion
 
 __all__ = ['excise_regions', 'linear_exciser', 'spectrum_from_model']
 
+def true_exciser(spectrum, region):
+    """
+    Basic spectral excise method where the array elements in the spectral
+    region defined by the parameter ``region`` (a `~specutils.SpectralRegion`)
+    will be deleted from all applicable elements of the Spectrum1D object:
+    flux, spectral_axis, mask, and uncertainty. If multiple subregions are
+    defined in ``region``, all the subregions will be excised.
+
+    Other methods could be defined by the user to do other types of excision.
+
+    Parameters
+    ----------
+    spectrum : `~specutils.Spectrum1D`
+        The `~specutils.Spectrum1D` object to which the excision will be applied.
+
+    region : `~specutils.SpectralRegion`
+        The region of the spectrum to remove.
+
+    Returns
+    -------
+    spectrum : `~specutils.Spectrum1D`
+        Output `~specutils.Spectrum1D` with the region excised.
+
+    Raises
+    ------
+    ValueError
+       In the case that ``spectrum`` and ``region`` are not the correct types.
+
+    """
+
+    spectral_axis = spectrum.spectral_axis
+    excise_indices = None
+
+    for subregion in region:
+        # Find the indices of the spectral_axis array corresponding to the subregion
+        wavelengths_in = (spectral_axis >= region.lower) & (spectral_axis < region.upper)
```

Comment:
I know this isn't particular to this PR, but something we certainly should have caught when this was implemented. The spectral axis is not guaranteed to be in wavelength space, so I think we ought to avoid using the term "wavelengths".

```suggestion
        region_mask = (spectral_axis >= region.lower) & (spectral_axis < region.upper)
```